internal/controller/stas/workload_controller.go

```diff
@@ -149,6 +149,8 @@ func (r *PodReconciler) reconcile(ctx context.Context, pod *corev1.Pod) error {
 			cis.Spec.Image = image.Image
 			cis.Spec.Tag = image.Tag
 
+			// Ensure MinSeverity unset until we eventually make use of it
+			cis.Spec.MinSeverity = nil
```

Comment:
Now seems related to line below. Move it to grouping above or have it in a separate group?

```suggestion
			// Ensure MinSeverity unset until we eventually make use of it
			cis.Spec.MinSeverity = nil
```
```suggestion

			// Ensure MinSeverity unset until we eventually make use of it
			cis.Spec.MinSeverity = nil

```

Reply:
Moved it to the group above. PTAL!